authority/provisioner/acme.go

```diff
@@ -392,3 +442,12 @@ func (p *ACME) IsAttestationFormatEnabled(_ context.Context, format ACMEAttestat
 func (p *ACME) GetAttestationRoots() (*x509.CertPool, bool) {
 	return p.attestationRootPool, p.attestationRootPool != nil
 }
+
+// IsRootRevoked return a true if the serialNumber is part of the list
+// It will also be in charge of updating the list periodically if no CRL list is provided at configuration.
+func (p *ACME) IsRootRevoked(serialNumber string) bool {
+	if slices.Contains(p.AttestationFormats, "android-key") && !p.androidCRLTimeout.IsZero() && time.Now().After(p.androidCRLTimeout) {
+		p.initializeAndroidCRL()
+	}
+	return len(p.RootCRLs) > 0 && slices.Contains(p.RootCRLs, serialNumber)
```

Comment:
It's sufficient to just check using `Contains`, as it's backed by `range`:
```suggestion
	return slices.Contains(p.RootCRLs, serialNumber)
```